### PR TITLE
correct timestamps in standard destination tests and assume millis input in BQ

### PR DESCRIPTION
**Pull request by sherifnada**
## What
#963 introduced a bug by assuming the `emitted_at` fields passed to the BQ destination was in seconds, not millis. This PR reverts to the correct behavior, and fixes the timestamps in the standard destination dataset which led to the confusion. 

The issue in #963 was not caught since standard tests do not validate the correct `emitted_at` field was persisted, so I created #980 to address this. Also added #979 to improve the story around timetsamps to prevent such confusions in the future.

### CODE REVIEW

File: airbyte-integrations/connectors/destination-postgres/src/test-integration/java/io/airbyte/integrations/destination/postgres/PostgresIntegrationTest.java [hunk 1]
```diff
@@ -74,14 +75,10 @@ protected JsonNode getFailCheckConfig() {
...
   protected List<JsonNode> retrieveRecords(TestDestinationEnv env, String streamName) throws Exception {
-    return Databases.createPostgresDatabase(db.getUsername(), db.getPassword(), db.getJdbcUrl()).query(
-        ctx -> ctx
-            .fetch(String.format("SELECT * FROM \"%s\" ORDER BY emitted_at ASC;", NamingHelper.getRawTableName(streamName)))
-            .stream()
-            .map(r -> r.formatJSON(JSON_FORMAT))
-            .map(Jsons::deserialize)
-            .map(r -> Jsons.deserialize(r.get(COLUMN_NAME).asText()))
-            .collect(Collectors.toList()));
+    return retrieveRecordsFromTable(NamingHelper.getRawTableName(streamName))
```
Comment (sherifnada) [inline airbyte-integrations/connectors/destination-postgres/src/test-integration/java/io/airbyte/integrations/destination/postgres/PostgresIntegrationTest.java:78]: unrelated refactor

File: airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java [hunk 2]
```diff
@@ -269,7 +270,7 @@ public void acceptTracked(AirbyteMessage message) {
         final JsonNode data = Jsons.jsonNode(ImmutableMap.of(
             COLUMN_AB_ID, UUID.randomUUID().toString(),
             COLUMN_DATA, Jsons.serialize(message.getRecord().getData()),
-            COLUMN_EMITTED_AT, message.getRecord().getEmittedAt()));
+            COLUMN_EMITTED_AT, Instants.toSeconds(message.getRecord().getEmittedAt())));
```
Comment (michel-tricot) [inline airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java:273]: You're positive it is seconds and not milliseconds ?

**Review by michel-tricot — APPROVED** [review]

File: airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java (hunk 2 above)
Comment (sherifnada) [reply airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java]: sadface. Turns out BQ actually uses _microseconds_. https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#timestamp_type. I was confused because writing seconds produced the correct "human-formatted" timestamp on BQ when inspected by eye (like the screenshot below). 
Screen Shot 2020-11-14 at 6 47 34 PM

Goes to show the first thing I should have looked at was the docs before making changes. I changed this to use the official google libraries for date formatting and output as a timestamp-formatted string.
